index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```

Comment:
nee some space below tag
```suggestion
<head>
```

Reply:
Done